linux-tkg-config/6.5/minimal-modprobed.db

```diff
@@ -383,6 +389,7 @@ nls_utf8
 nouveau
 ntfs
 ntfs3
+nvidia_wmi_ec_backlight
```

Comment:
This one is also not in-tree.

Reply:
I don't get any warnings about this module, and it does build a `nvidia-wmi-ec-backlight.ko` file, so I think it is in-tree.

Reply:
Then I probably thought it wasn't there just by reading `nvidia` haha. All good then.